common/src/main/java/com/discordsrv/common/AbstractDiscordSRV.java

```diff
@@ -769,7 +769,19 @@ protected List<ReloadResult> reload(Set<ReloadFlag> flags, boolean initial) thro
                 messagesConfigManager().reload(configUpgrade, anyMissingOptions, backupPath);
 
                 if (anyMissingOptions.get()) {
-                    logger().info("Use \"/discordsrv reload config_upgrade\" to write the latest configuration");
+                    if (config().automaticConfigurationUpgrade) {
+                        if (configUpgrade) {
+                            logger().info("Tried to upgrade configuration, but some options are still missing.");
+                        } else {
+                            logger().info("Some configuration options are missing, attempting to upgrade configuration...");
+                            Set<ReloadFlag> reloadFlags = Collections.unmodifiableSet(Arrays.stream(ReloadFlag.values()).collect(Collectors.toSet()));
+                            return reload(reloadFlags, initial); // Retry with all flags including config upgrade
```

Comment:
Using all reload flags here is not ok. 

I'm not a fan of reloading inside reload, I would rather see something like:
```java
if (config() == null) {
    try {
        configManager().reload(false, new AtomicBoolean(false), null);
    } catch (Exception ignored) {}
}

configUpgrade = configUpgrade || (config() != null && config().automaticConfigurationUpgrade);
```